OHIF/Viewers
# Fix/buffer undefined (Issue 3588)

### Context

For Issue 3588: Changes in webpack config may have caused Buffer to be undefined when using datasetToBlob function from dcmjs dependency.

Applying buffer definition under plugins appears to be a solution:
https://stackoverflow.com/questions/68707553/uncaught-referenceerror-buffer-is-not-defined

https://github.com/webpack/changelog-v5/issues/10

### Changes & Results
Added buffer definition under webpack config.plugins allows datasetToBlob to work again.



### Testing


1. Start viewer
2. Select Water Phantom study (or any with CT and PT) in TMTV mode
3. Create "New Label" on right right
4. Select "Rectangle ROI Threshold Tool"
5. Create arbitrary rectangle in axial PT viewport
6. Click "Run"
7. Click "Create RT Report"

RT report should be prepared and prompt to save show up.

### Checklist

#### PR



- [] My Pull Request title is descriptive, accurate and follows the
  semantic-release format and guidelines.

#### Code

- [] My code has been well-documented (function documentation, inline comments,
  etc.)

#### Public Documentation Updates



- [] The documentation page has been updated as necessary for any public API
  additions or removals.

#### Tested Environment

- [] OS:  Windows 10
- [] Node version:  18.16.0
- [] Browser:
   Edge v115.0.1901.188


[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch
